### DISCUSSION TITLE
Why can't I run a test that mocks 'useStorage' like in the documentation?

### DISCUSSION THREAD

**Discussion by Kobee1203** [danielroe/nuxt-vitest#50] [Q&A]
- Nuxt: 3.1.1
- nuxt-vitest: 0.6.3

I followed the documentation to test the library.
I created a test file that mocks 'useStorage'.
When I run the tests, I get an error:

`Error: Cannot find import "useStorage" to mock`

Do I need to configure anything else?

Here is the example on Stackblitz:
https://stackblitz.com/edit/nuxt-starter-vkkrkv?file=tests/mytest.nuxt.test.ts

**Comment by Kobee1203** [comment]
I upgraded the nuxt-vitest version to 0.6.5, but I still have the same error.

What is missing to make it work?

**Reply by patrickyue** [reply]
have you tried using the example with vi.hoist, check the github homepage please
